internal/apps/apps.go

```diff
@@ -16,18 +21,33 @@ func hasValidExtension(file string, exts []string) bool {
 	return false
 }
 
-// IsStorageID checks if a link is an entry of app-storage.
-func IsStorageID(link string) bool {
-	re := regexp.MustCompile("^(storage:(//)?)?[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$")
-	if re.MatchString(link) {
+// IsStorageReference checks if a link is an entry of app-storage.
+func IsStorageReference(link string) bool {
+	if reFileID.MatchString(link) {
+		return true
+	}
+	if reFilePattern.MatchString(link) {
 		return true
 	}
 	return false
```

Comment:
If it's just those two, I'd suggest shortening it:

```suggestion
	return reFileID.MatchString(link) || reFilePattern.MatchString(link)
```